modules/embree/4.3.0/patches/bazel_build.patch

```diff
@@ -0,0 +1,658 @@
+--- BUILD.bazel
++++ BUILD.bazel
+@@ -0,0 +1,655 @@
++"""
++    SPDX-FileCopyrightText: 2023 Julian Amann <dev@vertexwahn.de>
++    SPDX-License-Identifier: Apache-2.0
++"""
++
++package(
++    default_visibility = ["//visibility:public"],
++)
++
++config_setting(
++    name = "windows_x86_64",
```

Comment:
This won't work with toolchain resolution, which is enabled by default in Bazel 7.

Reply:
Does a select with `"@platforms//os:windows"` work here, i.e. replacing

```
+}) + select({
+    ":windows_x86_64": [
+        "__SSE__",
+        "__SSE2__",
+        "EMBREE_STATIC_LIB",
+    ],
+    "//conditions:default": [
+    ],
+})
```

by

```
+}) + select({
+    "@platforms//os:windows": [
+        "__SSE__",
+        "__SSE2__",
+        "EMBREE_STATIC_LIB",
+    ],
+    "//conditions:default": [
+    ],
+})
```

?

Reply:
Switched to @platforms//os:windows